stories/authentication/sessionProvider.stories.tsx

```diff
@@ -63,8 +63,17 @@ export default {
 export function ProviderWithHook(): ReactElement {
   const [idp, setIdp] = useState("https://inrupt.net");
 
+  const restoreCallback = (url: string) => {
+    console.log(`Restoring page ${url}`);
+  };
+
   return (
-    <SessionProvider sessionId="session-provider-example" onError={console.log}>
+    <SessionProvider
+      sessionId="session-provider-example"
+      onError={console.log}
+      restorePreviousSession
```

Comment:
Was this the result of some ESLint rule? Otherwise, I always get confused by this if it's not explicit :P 

```suggestion
      restorePreviousSession={true}
```

Reply:
Yeah, it's the result of linting, I didn't even know it was possible until ESLint complained about it ;)